server/lib/util.py

```diff
@@ -120,3 +120,30 @@ def parse_date(date_string):
     return datetime.strptime(date_string, "%Y-%m-%d")
   else:
     raise ValueError("Invalid date: %s", date_string)
+
+
+# Returns a compact version of observation point API results
+def compact_point(point_resp, all_facets):
```

Comment:
why need to move the function here?

Reply:
Because this PR uses it from server/lib/nl/utils.py